odoo/addons/base/models/ir_model.py

```diff
@@ -697,11 +697,20 @@ def write(self, vals):
                 field = getattr(obj, '_fields', {}).get(item.name)
 
                 if vals.get('name', item.name) != item.name:
-                    # We need to rename the column
+                    # We need to rename the field
                     item._prepare_update()
-                    if column_rename:
-                        raise UserError(_('Can only rename one field at a time!'))
-                    column_rename = (obj._table, item.name, vals['name'], item.index)
+                    if item.ttype == 'one2many':
+                        # the one2many is not explicit in the database!
+                        pass
+                    elif item.ttype == 'many2many':
+                        if not item.relation_table:
```

Comment:
Is it possible to have no relation table on a custom m2m field? If it is, what does it imply?